controllers/app.controllers.js

```diff
@@ -1,9 +1,20 @@
-const { retrieveAllTopics } = require('../models/app.models.js')
+const { retrieveAllTopics, retrieveArticlesById } = require('../models/app.models.js')
 
 exports.getAllTopics = (req, res, next) => {
     retrieveAllTopics().then((topics) => {
         res.status(200).send(topics);
     })
         .catch(next);
 
+}
+
+exports.getArticlesById = (req, res, next) => {
+    const { article_id: id } = req.params;
+
+    retrieveArticlesById(id).then((article) => {
+        res.status(200).send({ articles: article })
+    })
```

Comment:
As this is a single article, I feel ```article``` is a suitable name for the key instead of articles plural